test/e2e/multihoming.go

```diff
@@ -67,6 +67,18 @@ var _ = Describe("Multi Homing", func() {
 			netConfig.namespace = f.Namespace.Name
 			podConfig.namespace = f.Namespace.Name
 
+			By("applying ovs bridge mapping")
```

Comment:
bridge-mapping is only relevant for localnet, so could we do it only on these cases? 
Also, could we DeferClean it after?

Reply:
since we just add a mapping to the default one, it is fine to do it always + to not clean it

Reply:
yeah man, let's only do this once needed, an strive to clean it up afterwards. 

You have enough time to provide amazing code. Let's take that opportunity. No pressure.